### PR TITLE
Empty collection text: Add link to parent category

### PR DESCRIPTION
**Pull request by jyee27**
Add a link to the parent category with the text `Return to parent category` on empty collection pages.
This replaces the "Try to navigate to subcategories..." text that was there previously.

Closes: #460

### CODE REVIEW

File: frontend/components/product-list/sections/FilterableProductsSection/index.tsx [hunk 1]
```diff
@@ -236,9 +248,13 @@ const ProductListEmptyState = forwardRef<EmptyStateProps, 'div'>(
                   Empty collection
                </Text>
                <Text maxW="500px" color="gray.500" textAlign="center" px="2">
-                  This collection does not have products. Try to navigate to
-                  subcategories to find what you are looking for.
+                  This collection does not have products.
                </Text>
+               {parentCategory && (
+                  <Link href={parentCategory.path}>
+                     Return to parent category
```
Comment (sterlinghirsh) [inline frontend/components/product-list/sections/FilterableProductsSection/index.tsx:255]: Can we use the title here?
```suggestion
                     Return to {parentCategory.title}
```

**Comment by erinemay** [comment]
@jyee27 @sterlinghirsh, I'm having trouble organically finding an empty collection. Any idea how to get to an empty page now?

I made `/Parts/Kindle_Tablet/Accessories` ([this branch](https://react-commerce-prod-o8r19m325-ifixit.vercel.app/Parts/Kindle_Tablet/Accessories) and [production](https://www.ifixit.com/Parts/Kindle_Tablet/Accessories)), but it doesn't seem to have the original text or the new text from this pull.

**Comment by jyee27** [comment]
@erinemay Try `/Parts/Android` for an empty collection.

**Comment by erinemay** [comment]
> @erinemay Try `/Parts/Android` for an empty collection. This one has a parent category that isn't the top level: `/Parts/Motorola_Moto_G5s_Plus`

QA 🗡️ Those worked. Checked both, and both have links to their parent category. Links go to the appropriate parent category collection. Link hover behavior (underline) is appropriate. 

image
image

**Comment by sumukhipandey** [comment]
CR 🦋

**Review by jarstelfox — APPROVED** [review]
CR 🌵 It looks like github requires an official approval to click the button

**Pull request merged**